src/Command/PqCommands/ContentHubPqLanguages.php

```diff
@@ -0,0 +1,182 @@
+      );
+    }
+
+    $incorrectParaConfigFields = $this->checkMultilingualParagraphConfiguredCorrect();
+    $kriName = 'List of incorrectly configured referenced paragraph fields.';
+    if (!empty($incorrectParaConfigFields)) {
+      $kriValue = '';
+      foreach ($incorrectParaConfigFields as $incorrectParaFields) {
+        $kriValue .= $incorrectParaFields . PHP_EOL;
+      }
+      $result->setIndicator(
+        $kriName,
+        trim($kriValue),
+        PqCommandResultViolations::$paragraphConfiguration,
+        TRUE,
+      );
+    }
+    else {
+      $result->setIndicator(
+      $kriName,
+      '',
+      'No incorrectly configured referenced paragraph fields.'
+      );
+    }
+
+    return 0;
+  }
+
+  /**
+   * Check enabled languages and translations count.
+   *
+   * @return array
+   *   Array of enabled languages and count of translations.
+   */
+  protected function getEnabledLanguagesData(): array {
+    $languagesData = [];
+    /** @var \Drupal\Core\Language\LanguageManagerInterface $languageManager */
+    $languageManager = $this->drupalServiceFactory->getDrupalService('language_manager');
+    $langcodeList = array_keys($languageManager->getLanguages());
+
+    /** @var \Drupal\Core\Entity\EntityTypeManagerInterface $entityTypeManager */
+    $entityTypeManager = $this->drupalServiceFactory->getDrupalService('entity_type.manager');
+    /** @var \Drupal\Core\Database\Connection $databaseManager */
+    $databaseConnection = $this->drupalServiceFactory->getDrupalService('database');
+
+    $entityTypes = $entityTypeManager->getDefinitions();
+    foreach ($entityTypes as $type) {
+      if ($type->entityClassImplements(ContentEntityInterface::class) && $type->isTranslatable()) {
+        foreach ($langcodeList as $langCode) {
+          $count = $databaseConnection->select($type->getDataTable())
+            ->condition('langcode', $langCode)
+            ->countQuery()
+            ->execute()
+            ->fetchField();
+          $languagesData[$langCode] += $count;
+        }
+      }
+    }
+    return $languagesData;
+  }
+
+  /**
+   * Checks if multilingual paragraphs are configured properly.
+   *
+   * @return array
+   *   Array of incorrect configured paragraph fields
+   */
+  protected function checkMultilingualParagraphConfiguredCorrect(): array {
+    /** @var \Drupal\Core\Entity\EntityTypeBundleInfoInterface $bundleInfo */
+    $bundleInfo = $this->drupalServiceFactory->getDrupalService('entity_type.bundle.info');
+    /** @var \Drupal\Core\Entity\EntityFieldManagerInterface $fieldManager */
+    $fieldManager = $this->drupalServiceFactory->getDrupalService('entity_field.manager');
+
+    $configuration = [];
+    $fieldMap = $fieldManager->getFieldMapByFieldType(self::ENTITY_REF_REV);
+    foreach ($fieldMap as $type => $fields) {
+      // Skipping reference revision fields inside paragraphs.
+      if ($type == self::PARAGRAPH_FIELD) {
+        continue;
+      }
```

Comment:
Let's not skip these fields as nested paragraphs with incorrect multilingual settings can also lead to wrong data.